src/main/java/org/folio/innreach/domain/service/impl/ContributionValidationServiceImpl.java

```diff
@@ -104,6 +104,11 @@ public boolean isEligibleForContribution(UUID centralServerId, Item item) {
       return false;
     }
 
+    if(isExcludedLocation(centralServerId, item)) {
+      log.info("Item's location is excluded from contribution");
```

Comment:
Consider logging the item id or similar to make the log message more useful

Reply:
Done